src/_pytest/terminal.py

```diff
@@ -245,6 +245,8 @@ def __init__(self, config, file=None):
         self.isatty = file.isatty()
         self._progress_nodeids_reported = set()
         self._show_progress_info = self._determine_show_progress_info()
+        self.rewrite_lines = 0
+        self.width_adjust = 1 if sys.platform == "win32" else 0
```

Comment:
can this be a global?

Reply:
`width_adjust` could certainly be global since the platform won't change while the code is running.  Is there a benefit to making it global?

Reply:
i'd like to see it as constant, since its clearly not something thats even remotely related to a lifecycle of a terminalwriter

Reply:
@RonnyPfannschmidt would you be OK with a class variable, or do you expect this to be at module scope?

Reply:
both would be fine, please ensure it follows the naming style for constants